WF-768: document command change (#27)
spatialos/FlexibleProjectExample: docs/reference/launch-configuration-with-the-Platform-SDK.md

> # Starting an FPL project using the Platform SDK

Reviewer: ```suggestion
# Starting a deployment using the Platform SDK
```

The [flexible project layout (FPL) launch configuration file](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/launch-configuration.md) has a different format from the [current project layout (CPL) launch configuration file](https://docs.improbable.io/reference/13.6/shared/reference/file-formats/launch-config#launch-configuration-file). You can find more details about the difference between the two formats [here](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/migration-guide/configs-conversion-guide.md#converting-the-launch-configuration-file).

The Platform SDK currently only works with the CPL launch configuration format, so you need to convert your launch configuration file back to CPL format before you can use it with the Platform SDK. We’ve added a command that does this for you.

## How do I use it?
To convert an FPL launch configuration file to its CPL equivalent, run the following command inside the directory containing your FPL-format `spatialos.json`:
```bash
spatial alpha config convert
```

> By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:

Reviewer: By default, we will create a file called `cpl_launch.json` containing the converted launch configuration.

Author: done

```bash
spatial alpha config convert --output_path <output-path>
```

> You can now use this file with the Platform SDK.

Reviewer: Can we put a link here to any existing docs?

Reviewer 2: Good plan:

> `You can now use this file with the [Platform SDK](https://docs.improbable.io/reference/latest/platform-sdk/introduction).`

Author: done

> **Notes:** The current workflow is temporary, and this command is only available in `spatial alpha`. We will have a better workflow in the future.

Reviewer: Put this higher up, e.g. in line 4.

Reviewer 2: Sorry @frozenL, I know I suggested moving it to the bottom! I don't mind where it goes - it's not crucial to the user at any particular point.

Author: no worries! done

# Starting a deployment using the Platform SDK

**Note:** The current workflow is temporary, and this command is only available in `spatial alpha`. We will have a better workflow in the future.

> The [flexible project layout (FPL) launch configuration file](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/launch-configuration.md) has a different format from the [current project layout (SPL) launch configuration file](https://docs.improbable.io/reference/13.6/shared/reference/file-formats/launch-config#launch-configuration-file). You can find more details about the difference between the two formats [here](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/migration-guide/configs-conversion-guide.md#converting-the-launch-configuration-file).

Reviewer: ```suggestion
The [flexible project layout (FPL) launch configuration file](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/launch-configuration.md) has a different format from the [current project layout (SPL) launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#launch-configuration-file). You can find more details about the difference between the two formats [here](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/migration-guide/configs-conversion-guide.md#converting-the-launch-configuration-file).
```

## How do I use it?
To convert an FPL launch configuration file to its CPL equivalent, run the following command inside the directory containing your FPL-format `spatialos.json`:
```bash
spatial alpha config convert
```
By default, this prints the result to `cpl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the CPL launch configuration file:
```bash

> spatial alpha config convert --output_path "<output_path>"

Reviewer: Nit: Just to be consistent with our existing code snippets:
```suggestion
spatial alpha config convert --output_path <output-path>
```

```
You can now use this file with the Platform SDK.

> 

Reviewer: Unnecessary whiteline

Author: thanks! done

> By default, this prints the result to `spl_launch.json`. If you want to give the resulting file a different name or save it in a different place, you can set the `--output_path` flag to specify a filepath to the SPL launch configuration file:

Reviewer: By default, we will create a file called `cpl_launch.json` containing the converted launch configuration.

Author: done
